starport/cmd/network_chain_prepare.go

```diff
@@ -0,0 +1,58 @@
+package starportcmd
+
+import (
+	"strconv"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/starport/starport/pkg/cosmosaccount"
+	"github.com/tendermint/starport/starport/services/network"
+)
+
+// NewNetworkChainPrepare returns a new command to prepare the chain for launch
+func NewNetworkChainPrepare() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "prepare [launch-id]",
+		Short: "Prepare the chain for launch",
+		Args:  cobra.ExactArgs(1),
+		RunE:  networkChainPrepareHandler,
+	}
+
+	c.Flags().String(flagFrom, cosmosaccount.DefaultAccount, "Account name to use for sending transactions to SPN")
```

Comment:
Can we reuse this in other network commands as well by moving to a flagSetX func?